wallets/provider-solflare/src/index.ts

```diff
@@ -0,0 +1,61 @@
+import type {
+  CanSwitchNetwork,
+  Connect,
+  WalletInfo,
+} from '@rango-dev/wallets-shared';
+import type { BlockchainMeta, SignerFactory } from 'rango-types';
+
+import { Networks, WalletTypes } from '@rango-dev/wallets-shared';
+import Solflare from '@solflare-wallet/sdk';
+import { solanaBlockchain } from 'rango-types';
+
+import signer from './signer';
+
+const WALLET = WalletTypes.SOLFLARE;
+
+export const config = {
+  type: WALLET,
+};
+
+const walletInstance = new Solflare();
+
+export const getInstance = () => walletInstance;
```

Comment:
When Solflare extension is not installed, your code doesn't detect it. I think it's better to have same behavior like other wallets.

Reply:
Tnx, handled it by checking the window for `solflare` object